### PR TITLE
add randomSeed to queries

### PR DESCRIPTION
**Pull request by kjaym**
@sparadiso Add client support for #47

### CODE REVIEW

**Comment by sparadiso** [comment]
@kjaym is random seed ever used?

**Comment by kjaym** [comment]
@sparadiso Correct. Use goes elsewhere.

**Comment by sparadiso** [comment]
LGTM - will merge when build suceeds.

**Comment by kjaym** [comment]
@sparadiso The build is failing on some deploying issues, which involve codeship changes that we aren't going to tackle right now. The code _does_ compile without error.

File: src/main/java/io/citrine/jcc/search/core/result/MultiSearchResult.java [hunk 1]
```diff
@@ -17,8 +17,7 @@
  * @author Kyle Michel
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
-public abstract class MultiSearchResult<T extends BaseSearchResult<?>>
-        implements Iterable<MultiSearchResultElement<T>> {
+public class MultiSearchResult<T extends BaseSearchResult<?>> implements Iterable<MultiSearchResultElement<T>> {
```
Comment (sparadiso) [inline src/main/java/io/citrine/jcc/search/core/result/MultiSearchResult.java]: ?

Comment (kjaym) [inline src/main/java/io/citrine/jcc/search/core/result/MultiSearchResult.java]: Yes sorry I snuck that in. It should definitely be there.

Comment (sparadiso) [inline src/main/java/io/citrine/jcc/search/core/result/MultiSearchResult.java]: 👍  just confirming

**Pull request merged**